packages/nns/src/canisters/governance/response.converters.ts

```diff
@@ -133,6 +133,10 @@ const toNeuron = ({
   dissolveState: neuron.dissolve_state.length
     ? toDissolveState(neuron.dissolve_state[0])
     : undefined,
+  spawnAtTimesSeconds:
+    neuron.spawn_at_timestamp_seconds.length > 0
```

Comment:
can probably be simplified to `neuron.spawn_at_timestamp_seconds[0]` only

```
const tmp = [];
console.log(tmp[0]); // => undefined
```

Reply:
good point!

Reply:
done